docs/providers/documentation/signl4-provider.mdx

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
can you please add this page to `mint.json`?
![CleanShot 2024-01-24 at 11 28 26@2x](https://github.com/keephq/keep/assets/12069200/060f8134-d5cc-4523-b86f-ace354e60aca)

Reply:
Done.